htdocs/takepos/invoice.php

```diff
@@ -1347,6 +1347,9 @@ function SetNote() {
 print $hookmanager->resPrint;
 
 if (empty($_SESSION["basiclayout"]) || $_SESSION["basiclayout"] != 1) {
+	if ($conf->global->TAKEPOS_SHOW_SUBPRICE) {
```

Comment:
Must use getDolGlobalString() to be compatible with php8

Reply:
done